Porting/test-dist-modules.pl

```diff
@@ -11,11 +11,16 @@
 
 my $continue;
 my $separate;
+my $install;
 GetOptions("c|continue" => \$continue,
            "s|separate" => \$separate,
+           "i|install"  => \$install,
            "h|help"     => \&usage)
   or die "Unknown options\n";
 
+$separate
```

Comment:
We could use some inline comments that describe what the various command-line options to `Porting/test-dist-modules.pl`.  And since we're looking at this statement, why does the program not exit once `&usage` is called (or when I call the program with `--help`?

Reply:
I've added POD and usage() now does an exit().